### DISCUSSION TITLE
Is it possible to use Custom Publishers with Dockers?

### DISCUSSION THREAD

**Discussion by ryancurrah** [goreleaser/goreleaser#2397] [Q&A] [answered]
We want to use the `Custom Publishers` pipe which will use the `jfrog-cli` to publish all our artifacts including `Dockers`. Problem is the `Custom Publishers` requires the `id` of the artifact and I do not think `Dockers` has an `id`.

So is it possible to custom publish a docker image?

**Comment by caarlos0** [comment, marked as answer]
hey, I think that's definitely a good feature request... and fairly simple to implement.

Will convert this to an issue and work on it this weekend probably.

**Reply by caarlos0** [reply]
#2398

**Reply by caarlos0** [reply]
#2399
